### PR TITLE
[MIG][13.0] stock_picking_invoice_link: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
TT21687

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This can be updated with #623

**Comment by sergio-teruel** [comment]
Yes!!! Please

**Comment by pedrobaeza** [comment]
@manuelcalerosolis can you attend this?

**Review by chienandalu — COMMENTED** [review]
Is this still WIP?

**Review by carlosdauden — APPROVED** [review]

File: stock_picking_invoice_link/models/sale_order.py [hunk 1]
```diff
@@ -1,26 +1,15 @@
...
-            and not x.invoice_line_id
-            and not x.location_dest_id.scrap_location
-            and x.location_dest_id.usage == "customer"
-        ).mapped("picking_id").write({"invoice_ids": [(4, invoice_id)]})
-        return super(SaleOrderLine, self).invoice_line_create_vals(invoice_id, qty)
-
-    @api.multi
-    def _prepare_invoice_line(self, qty):
-        vals = super(SaleOrderLine, self)._prepare_invoice_line(qty)
+    def x_prepare_invoice_line(self):
```
Comment (chienandalu) [inline stock_picking_invoice_link/models/sale_order.py]: I guess this method goes away in favor of the one below...

**Review by chienandalu — CHANGES_REQUESTED** [review]
Functional test ok. Just a little remark:

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

File: stock_picking_invoice_link/models/sale_order.py (hunk 1 above)
Comment (sergio-teruel) [reply stock_picking_invoice_link/models/sale_order.py]: Yes... of course!!!!

**Comment by sergio-teruel** [comment]
@chienandalu Changes done!!

**Comment by sergio-teruel** [comment]
This PR already added (https://github.com/OCA/stock-logistics-workflow/pull/623)

**Review by chienandalu — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Sorry I will increase the coverage...

**Comment by sergio-teruel** [comment]
Ready!!! :smile:

**Comment by carlosdauden** [comment]
Ready to merge.

**Pull request merged**